drivers/ipm/ipm_cavs_idc.c

```diff
@@ -3,9 +3,6 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-
-#define DT_DRV_COMPAT intel_cavs_idc
```

Comment:
How are you removing this?  How can the driver still compile?

Reply:
it builds using devicetree macros.

Reply:
hmm, `DT_INST_IRQN` in cavs_idc_init expects DT_DRV_COMPAT, looks like this driver is not being built in CI, otherwise it would fail.

Reply:
Ok I'll have a look at it today at later time